### ISSUE TITLE
Should `Screen` become a first-class concept?

### ISSUE THREAD

**Issue by goenning** [aptabase/aptabase#60]
By having a TrackScreen method we could automatically find:
- most popular screens (per country/os/version as usual)
- time spent on each screen
- build funnels of user journey

To think about:
1. Can this however be achieve with events + custom dashboard?
2. Would this be useful for WebApps and Games (Unity/Unreal) too?

**Comment by Aichnerc** [comment]
+1

**Comment by EinfachHans** [comment]
I'm also a fan of having a trackScreen method directly within the sdk 👍🏼

**Comment by karlships** [comment]
Yes this would be cool, altough an improved events explorer would also do the job
